### PR TITLE
[PR3] [Channel] [Internal] Generate channel.py file

### PR DESCRIPTION
**Pull request by samchris007**
### What does this Pull Request accomplish?
- Generates channel.py file using **channel.py.mako**
- Uses the metadata (**attributes,py**) from **PR:** https://github.com/ni/grpc-device-scrapigen/pull/148 to generate **channel.py**.

### Why should this Pull Request be merged?
Implements [Task 2316151](https://dev.azure.com/ni/DevCentral/_workitems/edit/2316151): Generate all the channel classes in task modules

### CODE REVIEW

**Review by bkeryan — CHANGES_REQUESTED** [review]

File: generated/nidaqmx/_task_modules/channels/channel.py
Comment (zhindes) [reply generated/nidaqmx/_task_modules/channels/channel.py:193]: I agree, `chan.type`, `chan.description`, etc. are preferable.

**Review by zhindes — CHANGES_REQUESTED** [review]
Nothing to add other than what Brad already pointed out. Remove the `chan_` prefix from the names. I think once that is done, we'll have no deprecations.

File: generated/nidaqmx/_task_modules/channels/channel.py
Comment (samchris007) [reply generated/nidaqmx/_task_modules/channels/channel.py]: Got it. Removed prefix `chan_`

**Review by zhindes — APPROVED** [review]
I'll merge once Brad gives his approval

**Pull request merged**
